rubin_sim/site_models/scheduled_downtime_data.py

```diff
@@ -82,12 +81,12 @@ def read_data(self):
         database. However, an alternate database file can be provided. The alternate
         database file needs to have a table called *Downtime* with the following columns:
 
-        night
-            int : The night (from start of simulation) the downtime occurs.
-        duration
-            int : The duration (units=days) of the downtime.
-        activity
-            str : A description of the activity involved.
+        night: `int`
```

Comment:
Can you add a space between "night" and ":", e.g., 

```
night : `int`
```

Reply:
Yes, definitely - forgot the space.

Reply:
Ah! you are absolutely right! So, yeah, I don't know..

Reply:
Looks like it works, although maybe not the clearest thing ever (a markdown table would likely be better, but it's not obvious how to accomplish this). Fine for now.